serde-generate/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "serde-generate"
-version = "0.10.0"
+version = "0.11.0"
```

Comment:
This doesn't appear to be a breaking API change? maybe you can just do a point release?

Reply:
I wonder if adding `#pyre-strict` could trigger a chain reaction with more typing errors in python user code.

Reply:
I guess it depends if `breaking` is defined in terms of Rust compilation and behaviors only :)

Reply:
eh fair point :) I was strictly thinking in terms of the crate's API but i suppose "breaking" has other  implications as well.